settings/moonchild-signalk-ws-settings.json

```diff
@@ -0,0 +1,48 @@
+{
+        "vessel": {
+                "name"  : "Moonchild",
+                "brand" : "Catalina",
+                "type"  : "320",
+                "uuid"  : "self",
+
+                "dimensions": {
+                        "length": 9.9,
+                        "width": 3.6,
+                        "mast": 14.5,
+                        "depthTransducer": 0.6,
+                        "keel": 1.8
+                }
+        },
+    "mdns": true,
+	"interfaces": {
+    "rest": false,
+    "ws": true,
+    "bower": false,
```

Comment:
But this is a configuration file (and pull request) for node server, it doesn't have anything to do with your server?

Bower allows the server to serve html clients installed under bower_components.